lib/charms/layer/apache_spark.py

```diff
@@ -233,7 +356,11 @@ def start(self):
         # stop services (if they're running) to pick up any config change
         self.stop()
         # always start the history server, start master/worker if we're standalone
-        utils.run_as('ubuntu', '{}/sbin/start-history-server.sh'.format(spark_home), 'hdfs:///user/ubuntu/directory')
+        history_server_event_logs = '/tmp/spark-events'
```

Comment:
Let's use the self.dist_config.path('spark_events') here.  And let's use it in the hdfs case as well, eg:

```python
history_server_event_logs = self.dist_config.path('spark_events')
if (hdfs):
  history_server_event_logs = "hdfs://%s" % self.dist_config.path('spark_events')
```